src/sparseml/yolov5/helpers.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import logging
+import os
+
+from sparsezoo.v2.helpers import setup_model_directory
+
+
+__all__ = ["get_model_directory"]
+
+
+def get_model_directory(
+    output_dir: str,
+    training_outputs_dir: str,
+    model_file_torch: str,
+) -> None:
+    """
+    Takes the `training_outputs_dir`
+    (the directory where the pipeline saves its training artifacts),
+    and saves the training artifacts to `output_dir` in the `ModelDirectory` structure.
+
+    :param output_dir: The output path where the artifacts are saved
+        (adhering to the in `ModelDirectory` structure)
+    :param training_outputs_dir: The path to the existing directory
+        with the saved training artifacts
+    :param model_file_torch: name of the final .pth/.pt file to be saved
+    """
+    training_outputs_files = glob.glob(os.path.join(training_outputs_dir, "*"))
+    logs_path = [
+        file
+        for file in training_outputs_files
+        if os.path.basename(file).startswith("events.out.")
+    ]
+    for root_file in ["sample_inputs.tar.gz", "sample_outputs.tar.gz"]:
+        root_file_path = os.path.join(training_outputs_dir, root_file)
+        if not os.path.exists(root_file_path):
+            raise ValueError(
+                f"File {root_file_path} missing. To create this file, "
+                "make sure that the training script is being ran with"
+                "`--num_export_samples` argument."
+            )
+
+    model_onnx_path = os.path.join(
+        training_outputs_dir, "weights", model_file_torch.replace(".pt", ".onnx")
```

Comment:
what is the purpose here/what are the potential `onnx` names here?

Reply:
Very good question. This is something that also got me confused - so I decided to give more flexibility to the user.

When you look at the directory with the training artifacts:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/97082108/177913108-ce3ef265-ce09-4568-9dee-cc1ac28fb604.png">
There are two types of models in `/weights`: `best.pt` and `last.pt`. Being unsure whether the potential ML researcher would choose one of them by default, I gave a possibility to point at the one which should eventually make it to the `ModelDirectory`.  But maybe we are always choosing one over the other? Happy to get your input on that @bfineran .

Reply:
Good catch, however we shouldn't be exporting both. Either best or last will be exported to model.onnx, would recommend removing this functionality

Reply:
this is why I leave it as an argument, to choose `best `or `last` :D

Reply:
we should just be exporting `model.onnx` though right?

Reply:
as long as the deployment directory ends up with `model.onnx` (and can keep existing behavior outside of this) we should be good